src/BetterPhpDocParser/PhpDocInfo/PhpDocInfo.php

```diff
@@ -490,7 +490,7 @@ public function getConstFetchNodeClassNames(): array
     }
 
     /**
-     * @return string[]
+     * @return list<string>
      */
     public function getArrayItemNodeClassNames(): array
```

Comment:
Can you revert those? We use `string[]` by convention

Reply:
Sure. Reverted.